src/patternfly/components/DataList/data-list.scss

```
@@ -56,7 +58,7 @@
   --pf-c-data-list__item-content--md--PaddingBottom: var(--pf-global--spacer--lg);
 
   // Cell
-  --pf-c-data-list__cell--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-data-list__cell--PaddingTop: calc(var(--pf-global--spacer--lg) + (var(--pf-c-data-list--LineHeight) * 1px));
```

You shouldn't need the nested calc. Order of operations will multiply first.

```suggestion
  --pf-c-data-list__cell--PaddingTop: calc(var(--pf-global--spacer--lg) + var(--pf-c-data-list--LineHeight) * 1px);
```

---

This would be 24px + 1.5px = 25.5px. What's the significance of adding the line height as a pixel value?